python/tests/unittests/test_vqe.py

```diff
@@ -123,7 +123,7 @@ def test_vqe_two_qubit_float(optimizer, kernel_two_qubit_vqe_float,
         optimizer,
         parameter_count=1,
         argument_mapper=lambda parameter_vector: parameter_vector[0],
-        shots=100)
+        shots=-1)
```

Comment:
@amccaskey I changed this to `-1` to use the exact `exp-val` mode (matched existing behavior). To pass the test conditions in `shots` mode, we'll need a very large `shots` value. This will impact test time and test stability. Please let me know what you think.